src/orderbook.js

```diff
@@ -125,8 +128,12 @@ function OrderBook(remote,
     });
   }
 
-  function updateFundedAmountsWrapper(transaction) {
-    self.updateFundedAmounts(transaction);
+  function onTransactionWrapper(transaction) {
```

Comment:
If I use `bind` I will need to introduce new variable to keep `bind` result because I need to unsubscribe that function from event. So I've choose separate function as more descriptive.